schema.graphql

```diff
@@ -476,6 +571,11 @@ type MatchQuestionBuzz {
   """
   player: Player!
 
+  """
+  Details on the timing of the buzz.
+  """
+  buzz_position: BuzzPosition
+
```

Comment:
Other metadata commonly tracked for buzzes (and bonus parts): answer given (including any prompts – maybe an array? – or stock reasons like conferring or no answer), protested, protest resolved